src/node_crypto.cc

```diff
@@ -1644,9 +1743,12 @@ void SSLWrap<Base>::VerifyError(const FunctionCallbackInfo<Value>& args) {
     x509_verify_error = SSL_get_verify_result(w->ssl_);
   }
 
-  if (x509_verify_error == X509_V_OK)
+  if ((x509_verify_error == X509_V_OK) && CheckWhiteList(w->ssl_))
     return args.GetReturnValue().SetNull();
 
+  // CheckWhiteList might change x509_verify_error. Check it again.
+  x509_verify_error = SSL_get_verify_result(w->ssl_);
```

Comment:
The case when `is_ok` is false, all of the verify error becomes `X509_V_ERR_CERT_REVOKED`.